### DISCUSSION TITLE
Split the repository

### DISCUSSION THREAD

**Discussion by LeMyst** [richardthombs/scunpacked#68] [Ideas]
Hi,

What about splitting the repository in 4 different repository ?
* Loader
* website
* api
* k8s

To be honest, I'm only interested in the Loader part, and resolving merge conflicts every time I update from upstream is a bit tiring.

**Comment by richardthombs** [comment]
Yep, I have thought about doing this. How come you are getting merge conflicts from other parts of the site though? I would have thought that if you are only changing the loader then the website and other parts should merge cleanly? R.--Richard Thombsgearstone.uk From: MystSent: 26 December 2020 07:57To: richardthombs/scunpackedCc: SubscribedSubject: [richardthombs/scunpacked] Split the repository (#68) Hi,What about splitting the repository in 4 different repository ?Loaderwebsiteapik8sTo be honest, I'm only interested in the Loader part, and resolving merge conflicts every time I update from upstream is a bit tiring.—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Reply by LeMyst** [reply]
It's only because I removed all the website/api/k8s on my repository and everytime you update it I got some conflicts.
It's mainly a selfish request, but I think that can make sense to split the repository.
